notebooks/06_validate.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            type = report['anomaly_type']
</pre><p>Since <code>type</code> is the Python's built-in keyword, could we avoid using <code>type</code> as a variable name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>